pkg/packagekit/index.html

```diff
@@ -20,7 +20,7 @@
 <html>
 
 <head>
-  <title translate>Software Updates</title>
+  <title translate>Software updates</title>
```

Comment:
If we adopt the navigation as title case PF4 recommendation, then this should remain Title Case.